organization-names/words.txt

```diff
@@ -0,0 +1,21 @@
+# cspell-tools: keep-case no-split
+
+AIST
+ANYbotics
+ASAM
+autonomoustuff
```

Comment:
Also add these for `Autonomou` and it's GitHub ID?

```suggestion
autonomoustuff
AutonomouStuff
astuff
```

Reply:
By the way, where is `autonomoustuff` used?

Reply:
I don't know if lowercase word `autonomoustuff` is actually used, just copied from `autoware-spell-check-dict`

Reply:
https://github.com/search?q=org%3Atier4+autonomoustuff&type=code&expanded_query=org%3Atier4+autonomoustuff

`autonomoustuff` is used in some URLs and file names

Reply:
I see, thank you!